connectors/bitbucket-server/README.md

```diff
@@ -0,0 +1,7 @@
+# BitBucket Server Connector
```

Comment:
It's "Bitbucket", not "BitBucket". Please replace all instances of this.

Reply:
Changed ```BitBucket``` to ```Bitbucket``` in all the places..